File: python/purge_two_pages.py
#!/usr/bin/python3

"""
    purge_two_pages.py

    MediaWiki API Demos
    Demo of `purge` module: Sending post request to purge two or more pages
    MIT license
"""
import requests

S = requests.Session()

URL = "https://en.wikipedia.org/w/api.php"

PARAMS = {
    "action": "purge",
    "titles": "Main_Page|Nonexistent",
    "format": "json"
}

R = S.post(url=URL, params=PARAMS)
DATA = R.text

print(DATA)

File: javascript/purge_two_pages.js
/*  
    purge_two_pages.js
 
    MediaWiki API Demos
    Demo of `purge` module: Sending post request to purge two or more pages

    MIT license
*/

var request = require('request').defaults({jar: true}),
    url = "https://en.wikipedia.org/w/api.php";

function purge() {
    var params = {
        action: "purge",
        titles: "Main_Page|Nonexistent",
        format: "json"
    };

    request.post({ url: url, form: params }, function (error, res, body) {
        if (error) {
            return;
        }
        console.log(body);
    });
}

// Start the function
purge();